integration_test/third_party_apps_data/applications/postgresql/centos_rhel/install

```diff
@@ -1,11 +1,11 @@
 set -e
 
 source /etc/os-release
-case $VERSION_ID in
+case "${VERSION_ID}" in
   7)
     sudo yum -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-7-x86_64/pgdg-redhat-repo-latest.noarch.rpm
     ;;
-  8)
+  8|8.*)
```

Comment:
rocky linux 8 currently has version 8.6. The `8|8.*` form will support centos 8, (which is EOL but anyway), as well as any version starting with `8.`.